### PR TITLE
Feature/colormaps few colors

### PR DESCRIPTION
**Pull request by ResearchDaniel**
Added colormaps with one and two colors. 
Added processor progress unsubscribe command to webbrowser API

### CODE REVIEW

File: modules/webbrowser/src/processors/processorcefsynchronizer.cpp [hunk 1]
```diff
@@ -64,6 +64,7 @@ bool ProcessorCefSynchronizer::OnQuery(CefRefPtr<CefBrowser> browser, CefRefPtr<
     try {
         auto command = j.at("command").get<std::string>();
         auto subscribeProgressCommand = std::string("processor.progress.subscribe");
+        auto unsubscribeProgressCommand = std::string("processor.progress.unsubscribe");
```
Comment (petersteneteg) [inline modules/webbrowser/src/processors/processorcefsynchronizer.cpp]: const?

Comment (petersteneteg) [reply modules/webbrowser/src/processors/processorcefsynchronizer.cpp]: could use constexpr string_view for these string literals...

File: modules/webbrowser/src/processors/processorcefsynchronizer.cpp [hunk 2]
```diff
@@ -86,6 +87,19 @@ bool ProcessorCefSynchronizer::OnQuery(CefRefPtr<CefBrowser> browser, CefRefPtr<
             } else {
                 callback->Failure(0, "Processor " + p + " is not ProgressBarObservable");
             }
+        } else if (!command.compare(0, unsubscribeProgressCommand.size(),
```
Comment (petersteneteg) [inline modules/webbrowser/src/processors/processorcefsynchronizer.cpp]: why not just use ==

File: modules/webbrowser/src/processors/processorcefsynchronizer.cpp [hunk 3]
```diff
@@ -86,6 +87,19 @@ bool ProcessorCefSynchronizer::OnQuery(CefRefPtr<CefBrowser> browser, CefRefPtr<
...
+                                    unsubscribeProgressCommand)) {
+            auto p = j.at("path").get<std::string>();
+            auto network = parent_->getNetwork();
+            auto processor = network->getProcessorByIdentifier(p);
+            if (!processor) {
```
Comment (petersteneteg) [inline modules/webbrowser/src/processors/processorcefsynchronizer.cpp]: use if(auto processor = ... ){

**Review by petersteneteg — APPROVED** [review]

File: modules/webbrowser/src/processors/processorcefsynchronizer.cpp (hunk 1 above)
Comment (ResearchDaniel) [reply modules/webbrowser/src/processors/processorcefsynchronizer.cpp]: Fixed

File: modules/webbrowser/src/processors/processorcefsynchronizer.cpp (hunk 2 above)
Comment (ResearchDaniel) [reply modules/webbrowser/src/processors/processorcefsynchronizer.cpp]: Fixed, was too much copy-paste

**Pull request merged**